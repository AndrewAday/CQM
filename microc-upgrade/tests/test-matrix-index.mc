int main()
{
  fmatrix fm;
  int i;

  fm = init_fmat_const(2.45, 1, 6);

  for (i = 0; i < cols(fm); i = i + 1)
    print_float(fm[0,i]);

  return 0;
}
