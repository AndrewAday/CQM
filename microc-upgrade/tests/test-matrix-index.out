2.450000
2.450000
2.450000
2.450000
2.450000
2.450000
